### ISSUE TITLE
Add a book chapter discussing how to create open source communities that are diverse and inclusive 

### ISSUE THREAD

**Issue by lmaffeo** [open-organization/open-org-leaders-manual#34]
After years of talk about tech's lack of diversity, open source leaders want to learn how they can solve this problem. If OS maintainers design communities with diversity and inclusion embedded in their projects' strategies, many past problems will be easier to address. This proposed new chapter to the Leaders Manual will discuss:

- The difference between diversity and inclusion

- Why diverse, inclusive open source communities are more successful

- How to embed diversity and inclusion into your OS project's strategy

- Case studies of successfully diverse, inclusive OS communities, including Mozilla and Kubernetes

This chapter will be based on a presentation I gave at OS Summit North America in San Diego in August 2019.

**Comment by semioticrobotic** [comment]
Thanks for suggesting this chapter, @lmaffeo! I agree the Leaders Manual would benefit greatly from it. I'd like us to reflect on some questions of scope, target audience, and applicability before we jump into drafting and editing.

In part, the Open Organization project (and the *Open Organization* book series in particular) aims to demonstrate how adopting open principles can help practitioners create more innovative organizational cultures—even if those folks aren't necessarily working "in open source" or even in the software industry. Whenever possible, we aim to make the advice in our articles and books applicable to the largest possible audience. So I'm wondering if lessons from your chapter might be generalizable outside the context of open source community management to a larger, cross-industry audience.

For example, do you feel that diversity and inclusion are vital to the success of project teams in general, not just teams oriented toward the maintenance of open source projects? And have you seen open source communities experimenting with open principles and practices around D&I in ways that might serve as examples for other open organizations (or aspiring open organizations) to follow? Could experiments in embedding D&I into project strategies benefit these organizations, no matter their industry? How could we do it? And what, then, does that shift in strategy mean for leaders seeking to shift their own leadership philosophies and behaviors?

My goal here is to ensure that the widest possible audience can engage with your work—and in our case, the audience for the *Open Organization* book series is generally not engaged directly in open source development, open source community management, or the business of open source. Rather, they're coaches, consultants, people managers, executives, and other folks interested in the "people and process side" of open source and openness' ability to transform those facets of their organizations.

**Comment by lmaffeo** [comment]
Got it, thanks for the context @semioticrobotic! I first assumed that OS maintainers, Program Office leads, etc. are the book's key audience, so knowing that the content can go broader is a big help. 

I have lots of evidence to back the argument that diverse, inclusive teams yield better outcomes and financial returns, even though those teams don't always feel as harmonious as homogenous teams. If your goal is for all new book content to reach a wide audience, I have every confidence that I can make it happen :) 

Let me know the best next step and I'm on it!

**Comment by semioticrobotic** [comment]
> I have lots of evidence to back the argument that diverse, inclusive teams yield better outcomes and financial returns, even though those teams don't always feel as harmonious as homogenous teams. If your goal is for all new book content to reach a wide audience, I have every confidence that I can make it happen :)

And I, in turn, had every confidence you could, too! I'm very grateful for this contribution. Thanks, @lmaffeo.

The next best step is whatever is comfortable for you and your preferred writing process. If you'd like to share an outline, I'd be happy to review and help shape the chapter scope and terrain (we can do that here or in a separate document of your choice). Or if you'd prefer to just get down to it, you're welcome to draft something and share it whenever you'd like first round of editorial feedback (needn't be complete). Again, we can do that here or in a space of your choosing—whatever is comfortable for you.

I am currently in the process of "refactoring" the code in the book series repositories to make it the books themselves more amenable to community contributions and pull requests. As it stands now, we share different "binaries" in the repos, but I'd like to get us to a point where the chapters themselves are plain text, where people can work on them directly, and where new contributors can simply submit new chapters via pull request. Not quite there yet, but your request to write a new chapter certainly does give me some well-needed impetus. I am thankful for it!

In the meantime, though, we may need to go "old school" on this one and work/edit/collaborate in a separate space, then have me typeset and include the final version in an updated version of the book, pushed to the repo after we're finished.

Drop me a line with any questions, natch. Thanks again!

**Comment by semioticrobotic** [comment]
> OS maintainers, Program Office leads, etc.

By the way, @lmaffeo, if you are interested in writing on this topic for this audience in particular, then I certainly encourage you to pitch something to [The Open Source Way](https://github.com/theopensourceway/guidebook), for which these folks are indeed a target audience. If you'd like to explore that additional project, I'm happy to help there, too.

**Comment by lmaffeo** [comment]
@semioticrobotic I'm happy to work in here while we iron out the chapter details. To start, it would really help me to have a target page/word count - what's the standard for most chapters of this book, and what do you think works best for the audience? 

Having this scope in mind will really help me as I write the chapter's outline. Then we can move on to writing/editing in a separate doc :)

Also, are hard copies of the manual published when updated? If so, what's the timeline for that? 

Finally (for now), does the Manual follow a specific style guide (AP, Chicago, etc.)?

**Comment by semioticrobotic** [comment]
All excellent questions, @lmaffeo! Answers below.

> To start, it would really help me to have a target page/word count - what's the standard for most chapters of this book, and what do you think works best for the audience?

The project doesn't specify a standard length for chapters; however, since most of the piece began life as Opensource.com articles, they typically tend to hover somewhere in the 2,000 word range. More is fine, of course; your topic is a hefty one!

> Also, are hard copies of the manual published when updated? If so, what's the timeline for that?

Yes. We usually "push" new paper copies to the publisher after significant updates, and I consider a new chapter on D&I a significant update, so I will be submitting a new one! This usually doesn't take long. I am the bottleneck, not Lulu. But I can likely have this finished in a week or so after finalization.

> Finally (for now), does the Manual follow a specific style guide (AP, Chicago, etc.)?

No standard style! When an author cites or references something, we typically place the reference in a footnote (so we're closer to Chicago than AP!). But that's our strongest convention.

Hope that helps!

**Comment by lmaffeo** [comment]
Great, really helpful to know about the word count! Should I write and share an outline with you as the next step?

**Comment by semioticrobotic** [comment]
That sounds perfect. Let's do it!

**Comment by lmaffeo** [comment]
I'll share an outline by EOD Sunday, May 3rd :)

**Comment by lmaffeo** [comment]
Hey @semioticrobotic, thanks for your patience while I wrote the outline - had a hectic few weeks at work since we last spoke, but I'm ready to get on the writing train! My goal is to have a draft of the chapter written and turned in by EOD June, i.e. Q2. This is to keep myself accountable to the project, rather than letting it linger and procrastinating. 

The outline below is based on a talk I gave on this subject at OS Summit North America last August. Let me know how it looks; I'll start writing once I get the green light from you!

### Intro

- Discuss the range of initiatives aimed at bringing more women and minorities into tech.
- Pivot to discussing the disconnect between these opportunities and low rates of minority participation in tech, especially open source

### The state of open source diversity in tech

- Start by emphasizing that open source - which should theoretically be the most "meritocratic" tech ecosystem - is less diverse than tech at large, citing stats from Stack Overflow's latest survey. 
- Pivot to dismantling the "pipeline problem" myth that there aren't "enough" minorities with the skills required, explaining that the rates of minority CS graduates are higher than respective rates of CS employees. 

### The Diversity Paradox
- Using a case study, explain that when minorities do enter tech spaces, they often feel alien and marginalized. This causes them to leave, perpetuating the problem.
- Main point: If you haven't built an inclusive community, your diversity initiatives will fail.

### Diversity and Inclusion Aren't Synonyms

- They address different aspects of the same problem: Exclusionary Success

- Too often, diversity is a due diligence box for businesses to check

- But no one joins open source projects to check a box - they do it because, to quote W.E.B. DuBois, "We want to be creators in the kingdom of culture."

- Share some stats on how inclusive OS communities - those that put inclusion at their strategies' core - are more successful. Quantify "success" here in terms of PRs, number of releases, number of maintainers/contributors, etc.

- Reinforce this with stats from Deloitte where 80% of employees say inclusion matters when selecting employers, and 72% would leave one employer for a more inclusive one. 

- Conclusion: If your OS community isn't inclusive, you won't grow your contributor base and your project will lose relevance.

### Tips to attract and retain inclusive OS communities

1. Acknowledge your own bias

2. Audit your communities based on diversity (gender, POC, neurodivergence, etc.)

3. Seek out sources for new, diverse OS contributors, like CHAOSS/the Linux Foundation and Project Include

4. Give your community open and private options to leave feedback on their experiences. This can range from quarterly surveys to giving contributors the freedom to create channels in the project's Slack, Discourse, etc. chat about mental health, being a person of color, how to handle negotiations, etc.

5. Create and reward skills-based teams. 
Assign new contributors to senior maintainers
Use influence to promote proteges
Assign proteges to PRs that match their interests/expertise. This saves them hours of searching for open PRs on new, unfamiliar projects

### Conclusion
Take the steps above BEFORE trying to recruit diverse OS project contributors, because inclusive change starts from within.

**Comment by semioticrobotic** [comment]
I think this is an outline of a really great (and very important) chapter, @lmaffeo. As I understand it, the chapter you've outlined here will offer advice to open source practitioners, community managers and architects, etc., on creating more diverse and inclusive projects and teams. This is *slightly* out of scope for the *Open Organization* series, the audience for which is not open source practitioners but rather organizational leaders more generally (project and team leads, people managers, consultants, coaches, etc.) interested in better understanding how lessons and practices *from* open source projects and communities might be translated into more general advice for building open environments in their organizations (which needn't be software companies, nor open source communities themselves).

So what I might suggest, if you're open (har!) to it, is that you consider including this in the soon-to-relaunch [*Open Source Way* guidebook](https://github.com/theopensourceway/guidebook). That guide's target audience is **precisely** the one that seems most amenable to the message (and advice) you've outlined here, and your outline would be drop-in ready [as a pitch for that book](https://github.com/theopensourceway/guidebook/issues). I'd green-light it there without hesitation. This way, you could simply get to writing and we wouldn't risk violating the integrity of what is already a strong plan simply to make it "fit" a different audience.

The only reason I hesitate is that I know you'd expressed a desire to publish a chapter by the end of the calendar year, and I *believe* the current project timeline for *The Open Source Way* has us shipping a finished product in January or February 2021. But we'll be releasing "preview" chapters of the book early and often to generate interest in the work, and I can imagine we'd want to ship this among them as an example of the excellent work coming from the project.

Something to consider. If it's not appealing to you, we can surely work to refit the work a bit for the *Leaders Manual'*'s current target audience.

**Comment by lmaffeo** [comment]
Hey @semioticrobotic, thanks for the feedback! I think it sounds great and am happy to have it in the guidebook. Let me know which timelines you have in mind, and we'll take it from there :) Really excited to work on this with you!

**Comment by semioticrobotic** [comment]
Really glad to hear it, @lmaffeo! I'm excited, too!

I'd suggest the following steps

1. Open an issue in the Open Source Way repo with a title that mirrors other like "Write a chapter on..."
2. Add the chapter outline above :point_up: as a comment in that issue.
3. I'll see a notice that you've done so, and I'll work with you on getting the proposed chapter underway and aligning it in the book's working table of contents.

Then we should be all set!

**Comment by semioticrobotic** [comment]
Wait! I realize I can *transfer* an issue! Let me see if I have what it takes.

**Comment by semioticrobotic** [comment]
Still interested in pitching this to @theopensourceway, @lmaffeo? I know we'd still love to have it. If I can assist in any way, please just let me know.

**Comment by lmaffeo** [comment]
Hey @semioticrobotic sorry for the confusion! I definitely still want to write this chapter; was waiting on next steps since I thought you would transfer the issue, and didn't see your updated comment. Will follow the steps above now!

**Comment by lmaffeo** [comment]
[Done ](https://github.com/theopensourceway/guidebook/issues/101)

**Comment by semioticrobotic** [comment]
> Hey @semioticrobotic sorry for the confusion! I definitely still want to write this chapter; was waiting on next steps since I thought you would transfer the issue, and didn't see your updated comment. Will follow the steps above now!

Ah! I am so sorry about that. Hope my ambiguity did not delay your work too badly. I will get you all set up in the new place.

**Comment by semioticrobotic** [comment]
Closing this as it's now being incorporated into _The Open Source Way_.

**Comment by lmaffeo** [comment]
> Closing this as it's now being incorporated into _The Open Source Way_.

Sounds great @semioticrobotic, thanks for the update! Any edits needed on my end prior to publication? Do you have a rough estimate for when the chapter/book will be available?

**Comment by semioticrobotic** [comment]
You bet, @lmaffeo! _Open Source Way_ editors have your chapter on their radar. Someone from the editorial team will be assigned to it soon (might even be me!) and then you'll receive an update from the project.
